snprc_ehr/resources/queries/study/demographicsParentStatus.query.xml

```diff
@@ -9,11 +9,7 @@
                     </column>
                     <column columnName="Parent_id">
                         <columnTitle>Parent Id</columnTitle>
-                        <fk>
-                            <fkDbSchema>study</fkDbSchema>
-                            <fkTable>animal</fkTable>
-                            <fkColumnName>id</fkColumnName>
-                        </fk>
+                        <url>ehr/participantView.view?participantId=${Parent_id}</url>
```

Comment:
what if the Parent_id is not available or null Terry?

Reply:
Then there is no link to click on